extensions/amp-story/0.1/progress-bar.js

```diff
@@ -118,39 +133,53 @@ export class ProgressBar {
 
 
   /**
-   * @param {number} pageIndex The index of the new active page.
+   * @param {string} pageId The index of the new active page.
    * @public
    */
-  setActivePageIndex(pageIndex) {
-    this.assertValidPageIndex_(pageIndex);
+  setActivePageIndex(pageId) {
+    const progressBarIndex = this.pageIdMap_[pageId];
+    this.assertValidPageIndex_(progressBarIndex);
```

Comment:
seems the assertion is unnecessary anymore.

Reply:
removed